### PR TITLE
add remember_me_preference cookie

### PR DESCRIPTION
**Pull request by hobbitronics**
[IDP-105](https://itse.youtrack.cloud/issue/IDP-105) Set cookie to not check remember me box if user unchecks it when logging in so we don't default it checked again in the future.

[IDP-108](https://itse.youtrack.cloud/issue/IDP-108) Keep "Remember me" unchecked if the user unchecks it (at least during a single login)

### CODE REVIEW

File: modules/mfa/src/Auth/Process/Mfa.php [hunk 1]
```diff
@@ -808,6 +810,33 @@ public static function setRememberMeCookies(
         setcookie('c2', $expireDate, $expireDate, '/', null, $secureCookie, true);
     }
 
+    /**
+     * Clear remember_me cookies (c1 and c2)
+     */
+    public static function clearRememberMeCookies(): void
+    {
+        $secureCookie = Env::get('SECURE_COOKIE', true);
+        setcookie('c1', '', time() - 3600, '/', null, $secureCookie, true);
```
Comment (briskt) [inline modules/mfa/src/Auth/Process/Mfa.php:819]: just curious why you used `time() - 3600` and not just `time()`

**Review by briskt — APPROVED** [review]
good implementation

File: modules/mfa/src/Auth/Process/Mfa.php (hunk 1 above)
Comment (hobbitronics) [reply modules/mfa/src/Auth/Process/Mfa.php:819]: I guess it's considered best practice by some on [stacko](https://stackoverflow.com/questions/56722961/is-setting-cookie-expiration-time-time-3600-always-correct)

Comment (hobbitronics) [reply modules/mfa/src/Auth/Process/Mfa.php:819]: It's even in the [docs](https://www.php.net/manual/en/function.setcookie.php)

Comment (briskt) [reply modules/mfa/src/Auth/Process/Mfa.php:819]: Kinda seems like a "we've always done it that way" sort of thing. I can't figure out how it helps. If it's a time zone issue, shouldn't it be `time() - 3600 * 24`?

Comment (hobbitronics) [reply modules/mfa/src/Auth/Process/Mfa.php:819]: You're probably right, but time drift of the client is also mentioned. Not sure how common that is these days.

**Pull request merged**